src/DynamoApplications/StartupUtils.cs

```diff
@@ -196,14 +203,14 @@ public static void PreloadShapeManager(ref string geometryFactoryPath, ref strin
         /// for now, building an updatemanager instance requires finding Dynamo install location
         /// which if we are running on mac os or *nix will use different logic then SandboxLookup
         /// </summary>
-        /// <param name="hostName">Dynamo variation identified by host.</param>
+        /// <param name="info">Dynamo host analytics info.</param>
         /// <returns></returns>
-        private static IUpdateManager InitializeUpdateManager(string hostName = "")
+        private static IUpdateManager InitializeUpdateManager(HostAnalyticsInfo info)
```

Comment:
Do we still need InitializeUpdateManager(HostAnalyticsInfo) ?
or can we obsolete it and only used COnfig + DynamoModel ?

Reply:
We still need to initialize it but, we can remove the param. Since this is a private function, no need to obsolete